Configuration/Skimming/python/PDWG_EXODisappMuon_cff.py

```diff
@@ -0,0 +1,56 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+
+from Configuration.EventContent.EventContent_cff import AODSIMEventContent
+EXODisappMuonSkimContent = AODSIMEventContent.clone()
+EXODisappMuonSkimContent.outputCommands.append('keep *_hbhereco_*_*')
+EXODisappMuonSkimContent.outputCommands.append('keep *_horeco_*_*')
+EXODisappMuonSkimContent.outputCommands.append('keep *_csc2DRecHits_*_*')
+
+exoDisappMuonsHLT = hlt.hltHighLevel.clone(
+   throw = False,
+   andOr = True,
+   HLTPaths = [
+	"HLT_IsoMu*_v*"
+   ]
+)
+
+disappMuonsSelection = cms.EDFilter('DisappearingMuonsSkimming',
```

Comment:
Once you have the fillDescription method implemented in the filter this config can be cloned and simplified

Reply:
I've just created the fillDescriptions method, but I'm not sure how the process of cloning and simplifying the filter works.

Reply:
```suggestion
from Configuration.Skimming.disappearingMuonsSkimming_cfi.py import *
disappMuonsSelection = disappearingMuonsSkimming.clone()
```